### DISCUSSION TITLE
Add mixed type chart

### DISCUSSION THREAD

**Discussion by mafuuu-dev** [filamentphp/filament#5246] [Ideas]
**I propose to add mixed type chart**
<img width="245" alt="Снимок экрана 2022-12-14 в 06 26 23" src="https://user-images.githubusercontent.com/88625413/207474222-052da668-68a8-4cc4-b8a4-a01829c3b2c9.png">

**Now it is not in the list of available, but it is useful**
<img width="827" alt="Снимок экрана 2022-12-14 в 06 26 09" src="https://user-images.githubusercontent.com/88625413/207474237-a91dbb58-70fb-4e53-bf94-4ac4d6f68349.png">

**Comment by ArtMin96** [comment]
I think you can extend `Filament\Widgets\ChartWidget` and make your chart

**Reply by mafuuu-dev** [reply]
I agree with you. But then it won't be part of the package
